src/packages/core/protocol/account.ts

```diff
@@ -65,6 +74,27 @@ export class Account {
 	/** Public key HEX Buffer */
 	private publicKeyBuffer: Buffer = Buffer.from([]);
 
+	/* Gets the balance of an address (Testnet only for now) */
+	public static async getBalance(address: string): Promise<Object> {
+		if(isValidAddress(address)){
+			let balanceApi = new BalanceApi(APIServerTestNet);
+			return await balanceApi.balance(address);
+		} else {
+			throw new Error("Invalid Address Used!");
+		}
+	}
+
+	/* Gets the balance of this account (works for test net only) */
+	public async getBalance(): Promise<Object> {
+
+		if(isValidAddress(this.address)){
```

Comment:
please, do not repeat code, in this case you could call the static method from here as:
```
return Account.getBalance(this.address);
```

Reply:
Fixed